src/main/webapp/app/exam/manage/student-exams/student-exams.component.ts

```diff
@@ -117,7 +117,7 @@ export class StudentExamsComponent implements OnInit {
      */
     handleGenerateStudentExams() {
         // If student exams already exists, inform the instructor about it and get confirmations for re-creation
-        if (this.studentExams && this.studentExams.length) {
+        if (this.studentExams?.length) {
```

Comment:
```suggestion
        if (this.studentExams.length) {
```
The `?` should not be needed since it now definitely cannot be `undefined`.